src/app/core/not-found/not-found.component.ts

```diff
@@ -0,0 +1,11 @@
+import { Component } from '@angular/core';
+import { CategoryRoutingName } from '@app/shared/models';
+
+@Component({
+  selector: 'not-found',
+  templateUrl: './not-found.component.html',
+  styleUrls: ['./not-found.component.scss'],
+})
+export class NotFoundComponent {
```

Comment:
To pasuje do shared/components, to nie jest core :)

Reply:
OK, https://github.com/coi-gov-pl/gui-web-pomogamukrainie-forum/pull/137/files